spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -357,8 +357,26 @@ return Public Key text in one of the supported formats.
 The `mp.jwt.verify.issuer` config property allows for the expected value of the `iss`
 claim to be specified.  A MicroProfile JWT implementation must verify the `iss` claim of incoming JWTs is present and matches the configured value of `mp.jwt.verify.issuer`.
 
+## JWT and HTTP headers
 
-#### Mapping Properties to Environment Variables
+### Configuration Properties
+
+#### `mp.jwt.token.header`
+
+The `mp.jwt.token.header` configuration property allows to set up the header which is expected to contain a JWT token.
+
+MP JWT implementations are required to support `Authorization` (default) or `Cookie` configuration values.
+
+Support for other headers or alternative authentication schemes is optional.
+
+#### `mp.jwt.token.cookie`
+
+The `mp.jwt.token.cookie` configuration property allows to set up the Cookie name (default is `Bearer`) which is
+expected to contain a JWT token.
+
+This configuration will be ignored unless `mp.jwt.token.header` is set to `Cookie`.
+
```

Comment:
Hi @radcortez Referring to earlier point about providing some basic guidance, this is kind of out of scope how to create the secure cookies, but something simple like the following can be added here:
```
Providing the recommendations on how to securely use JWT tokens as a cookie is out of scope for the MP JWT specification but generally one should avoid putting a sensitive user information into a signed JWT token.
```

I think it is worth adding something like this to avoid some users saying MP JWT is careless in supporting the cookies. The issue of dealing with the cookies securely is not really a JWT related concern but providing a basic guideline would be fine I guess...

Reply:
Added!